lambda/resize.js

```diff
@@ -59,9 +66,21 @@ exports.default = async (rules, imageVehicle, storageKey, uuid, imageName, tempO
     let argsArray;
     if (imageName.split('.')[1] === 'gif') {
       argsArray = [tempOriginal, ...magickGifArgs, resizedPath];
+      // const [tempOriginalGif, ext] = tempOriginal.split('.');
+      // const tempGif = [tempOriginalGif, '-resized', '.', ext].join('');
+      // console.log('tempGif ', tempGif);
+      // childProcess.execFile(gifsicle, [tempOriginal, '-o', tempGif, ...gifsicleArgs], (error) => {
+      //   if (error) {
+      //     console.log('gifsicle err ', error);
+      //     return error;
+      //   }
+      //   argsArray = [tempGif, ...magickGifArgs, resizedPath];
+      //   return true;
+      // });
```

Comment:
is this supposed to be commented out?

Reply:
I have it commented out since it should work, but doesn't, and I don't have a commit for it.